src/lib/features/project-status/project-lifecycle-read-model/project-lifecycle-read-model-type.ts

```diff
@@ -0,0 +1,28 @@
+export interface IProjectLifecycleSummaryReadModel {
+    getProjectLifecycleSummary(
+        projectId: string,
+    ): Promise<ProjectLifecycleSummary>;
+}
+
+export type ProjectLifecycleSummary = {
+    initial: {
```

Comment:
I feel like this should be type

{
        averageDays: number | null;
        currentFlags: number;
    };

Reply:
Hmm? Not sure I understand what you mean? As in extract out those two fields into its own thing? Sure, we can do that. Archived is different, but that might be alright.

Reply:
If I understood you correctly, it's done in 28236f4491b419b544c708994b61c397383c0b4f